packages/terra-site/themeable-variables.json

```diff
@@ -426,6 +426,76 @@
     "--terra-table-row-selected-hover-background-color": "#cbe7fa"
   },
+  "Tabs": {
+    "--terra-tabs-modular-background-color": "#fff",
+    "--terra-tabs-collapsed-modular-tab-bar-border-color": "#dddedf",
+    "--terra-tabs-collapsed-modular-tab-bar-border-width": "1px",
+    "--terra-tabs-modular-color": "#007ee0",
+    "--terra-tabs-modular-tab-bar-border-color": "#dddedf",
+    "--terra-tabs-modular-tab-bar-border-width": "1px",
+    "--terra-tabs-modular-tab-bar-padding": "0",
+    "--terra-tabs-modular-extended-flex": "0 1 auto",
+    "--terra-tabs-modular-background-image": "linear-gradient(to bottom, #007ee0, #007ee0)",
+    "--terra-tabs-modular-background-position": "0 100%",
+    "--terra-tabs-modular-background-size": "0 2px",
+    "--terra-tabs-modular-active-background-size": "100% 2px",
+    "--terra-tabs-modular-active-color": "#000",
+    "--terra-tabs-modular-hover-background-image": "linear-gradient(to bottom, #c8cacb, #c8cacb)",
+    "--terra-tabs-modular-hover-background-size": "100% 2px",
+    "--terra-tabs-modular-hover-color": "#000",
+    "--terra-tabs-modular-disabled-color": "#c8cacb",
+    "--terra-tabs-modular-focus-background-color": "#fff",
+    "--terra-tabs-modular-focus-border-radius": "0",
+    "--terra-tabs-keyboard-focus-rtl-box-shadow": "0 0 1px 2px #4a90e2",
+    "--terra-tabs-keyboard-focus-ltr-box-shadow": "0 0 1px 2px #4a90e2",
+    "--terra-tabs-modular-active-focus-background-color": "#fff",
+    "--terra-tabs-modular-selection-bar-color": "#fff",
+    "--terra-tabs-modular-selection-bar-height": "0",
+    "--terra-tabs-modular-selection-bar-transition": "none",
+    "--terra-tabs-structural-content-background-color": "#fff",
+    "--terra-tabs-structural-content-border-radius": "0",
+    "--terra-tabs-structural-collapsed-menu-bar-border-color": "#dddedf",
+    "--terra-tabs-structural-collapsed-menu-bar-border-radius": "0",
+    "--terra-tabs-structural-tab-bar-border-color": "#dddedf",
+    "--terra-tabs-structural-tab-bar-border-width": "1px",
+    "--terra-tabs-structural-background-color": "#fff",
+    "--terra-tabs-structural-background-image": "linear-gradient(to bottom, #007ee0, #007ee0)",
+    "--terra-tabs-structural-background-position": "0 100%",
+    "--terra-tabs-structural-background-size": "0 2px",
+    "--terra-tabs-structural-border-color": "#dedfe0",
+    "--terra-tabs-structural-border-radius": "0",
+    "--terra-tabs-structural-border-width": "0 1px 0 0",
+    "--terra-tabs-structural-color": "#007ee0",
+    "--terra-tabs-structural-margin-between-tabs": "0",
+    "--terra-tabs-structural-transition-duration": "0",
+    "--terra-tabs-structural-transition-timing": "ease",
+    "--terra-tabs-structural-active-background-color": "#fff",
+    "--terra-tabs-structural-active-background-size": "100% 2px",
+    "--terra-tabs-structural-active-color": "#000",
+    "--terra-tabs-structural-disabled-background-color": "#fff",
+    "--terra-tabs-structural-disabled-color": "#909496",
```

Comment:
Lets update color from to `#909496` to `rgba(0,0,0,.25)` to better match inactive color used in other components